src/main/java/Model/Ticket.java

```diff
@@ -19,7 +25,8 @@ public class Ticket {
   private StadiumSectors stadiumSector;
   private BigDecimal ticketPrice;
 
-  public Ticket(String concertHall, int eventCode, long time, boolean isPromo, StadiumSectors stadiumSector, double maxBackpackWeight,
+
+  public Ticket(int classId, String concertHall, int eventCode, long time, boolean isPromo, StadiumSectors stadiumSector, double maxBackpackWeight,
       BigDecimal ticketPrice) {
 
     setConcertHall(concertHall);
```

Comment:
remove redundant setters

Reply:
Removed boiler-plate code, using lombok. Moved validation to a separate class.